lms/djangoapps/staticbook/views.py

```diff
@@ -100,7 +102,15 @@ def pdf_index(request, course_id, book_index, chapter=None, page=None):
         for entry in textbook['chapters']:
             entry['url'] = remap_static_url(entry['url'], course)
         if chapter is not None:
-            current_chapter = textbook['chapters'][int(chapter) - 1]
+            if int(chapter) <= (len(textbook['chapters'])):
+                current_chapter = textbook['chapters'][int(chapter) - 1]
+            else:
+                return redirect(
+                    reverse(
+                        pdf_index,
+                        kwargs={'course_id': course_id, 'book_index': book_index, 'chapter': 0}
```

Comment:
@adampalay I have addressed the comment. One thing to note here is that, providing invalid url in the address bar, will show the `pdfbook` chapter with the invalid chapter id in address bar.
E.g
pdfbook/book_id/chapter/100090
Will show chapter …. 100090 (in url)